src/main/java/ben/commands/EventCommand.java

```diff
@@ -0,0 +1,48 @@
+package ben.commands;
+
+import ben.storage.Storage;
+import ben.tasks.Event;
+import ben.tasks.Task;
+import ben.tasks.TaskList;
+import ben.ui.Ui;
+
+import java.time.LocalDate;
+
+/**
+ * Represents a command to add a new event task.
+ */
+public class EventCommand extends Command {
+    private final String description;
```

Comment:
I like that the fields (description, startDate, endDate) are declared as private and final, promoting encapsulation and information hiding.